compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -1236,6 +1236,99 @@ SmallVector<Range> UnPackOp::getIterationDomain(OpBuilder &builder) {
   return LinalgExt::getIterationDomain(*this, builder);
 }
 
+//===----------------------------------------------------------------------===//
+// Im2colOp
+//===----------------------------------------------------------------------===//
+
+SmallVector<Range> Im2colOp::getIterationDomain(OpBuilder &builder) {
+  Location loc = getLoc();
+  Value zero = builder.create<arith::ConstantIndexOp>(loc, 0);
+  Value one = builder.create<arith::ConstantIndexOp>(loc, 1);
```

Comment:
nit: let's start using `OpFoldResult`. I think using attribute is generally better than ConstantIndexOp. So it can be

```
OpFoldResult zero = builder.getIndexAttr(0);
OpFoldResult one = builder.getIndexAttr(1);
```